proco/connection_statistics/utils.py

```diff
@@ -157,3 +158,10 @@ def update_country_weekly_status(country: Country):
     country_status.avg_distance_school = country.calculate_avg_distance_school()
 
     country_status.save()
+
+
+def update_country_data_source_by_csv_filename(imported_file):
+    source = re.search(r'\d+-(.*)-\d+', imported_file.filename).group(1)
+    pretty_source = source.replace('_', ' ')
+    imported_file.country.data_source = pretty_source
+    imported_file.country.save()
```

Comment:
we'll need to keep multiple lines for data source, so don't replace it, just add line if not exists